src/frontend/apps/lti_site/components/LiveInfoBar/index.spec.tsx

```diff
@@ -1,11 +1,32 @@
-import { getDefaultNormalizer, screen } from '@testing-library/react';
+import { getDefaultNormalizer, screen, waitFor } from '@testing-library/react';
+import fetchMock from 'fetch-mock';
 import React from 'react';
 import { useParticipantsStore } from 'data/stores/useParticipantsStore/index';
 import render from 'utils/tests/render';
+import { report, useJwt } from 'lib-components';
 
 import { LiveInfoBar } from '.';
+import { wrapInVideo } from '../../utils/tests/wrapInVideo';
+import { videoMockFactory } from 'lib-components';
+import userEvent from '@testing-library/user-event';
```

Comment:
Should be moved at the top of the file and should be ordered with other  imports